assets/queries/openAPI/paths_object_empty/test/positive2.yaml

```diff
@@ -0,0 +1,11 @@
+openapi: 3.0.0
+info:
+  title: Simple API overview
+  version: 1.0.0
+paths:
+security:
```

Comment:
I think security and components can be removed from this example, since it is optional and does not affects query example...